README.md

```diff
@@ -36,3 +36,15 @@ production Slack bot would be.) To authorize on Slack:
   * Permissions: `channels:history`, `channels:read`, `chat:write:bot`, `reactions:read`, `users:read`
 * Install the app to the workspace (if it's not there) and add the OAuth token to the `slacktivate.cfg` in deployment.  (For SPC, do this through the Render UI.)
 * If you need to enable Dropbox auth: go to https://www.dropbox.com/developers/apps/create and create a new Dropbox API app with full access. Click "generate access token" on the app's config page and add that to `slacktivate.cfg` on Render.
+
+## Example Behavior
+
+Users invoke the slactivate bot by adding a :twitter: emoji or a :faq: emoji to a comment in slack.  These are custom emojis that can be added to slack.  
+
+Here's what the twitter emoji does: it creates a draft tweet in a separate slack channel #tweet-drafts.  Then the user can click a button there to have a tweet draft created in twitter in their browser, ready to be edited and tweeted.
```

Comment:
It's #tweetdrafts. (We should probably make this configurable but it's hardcoded right now.)